### PR TITLE
feat(learn): Add Stock Price Checker tests

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Related to #37185 

Adds *most* of the tests for the Stock Price Checker project. The one test I could not figure out was adding the likes - because it's restricted to once per IP, I wasn't sure how to ensure the test can run multiple times on the project successfully. 🤔 

Also, the `A good way to receive current prices is through our stock price proxy (replacing ''GOOG'' with your stock symbol): https://stock-price-checker-proxy--freecodecamp.repl.co/v1/stock/GOOG/quote` "test" doesn't really seem like a test. Should this be moved up into the challenge instructions instead?

### CODE REVIEW

File: curriculum/challenges/english/09-information-security/information-security-projects/stock-price-checker.english.md [hunk 1]
```diff
@@ -24,20 +24,55 @@ Start this project on Repl.it using <a href="https://repl.it/github/freeCodeCamp
...
+    testString: "async getUserInput => {
+      const data = await fetch(getUserInput('url') + '/api/stock-prices?stock=GOOG&stock=MSFT');
+      const parsed = await data.json();
+      const ticker = parsed.stockData;
+      assert.typeOf(ticker, 'array');
+      assert.property(ticker[0], 'rel_likes');
+      assert.property(ticker[1], 'rel_likes');
+    }"
   - text: 'A good way to receive current prices is through our stock price proxy (replacing ''GOOG'' with your stock symbol): https://stock-price-checker-proxy--freecodecamp.repl.co/v1/stock/GOOG/quote'
     testString: ''
```
Comment (moT01) [inline curriculum/challenges/english/09-information-security/information-security-projects/stock-price-checker.english.md:66]:
```suggestion
```

Yes, I would remove this @nhcarrigan. It is repeated in the instructions above, and probably shouldn't be a requirement to use it.

**Review by moT01 — COMMENTED** [review]
I vote to remove that user story. I suppose it would be fine to just leave it, but that's what I would do.

Other than that, this LGTM 🎉

File: curriculum/challenges/english/09-information-security/information-security-projects/stock-price-checker.english.md (hunk 1 above)
Comment (nhcarrigan) [reply curriculum/challenges/english/09-information-security/information-security-projects/stock-price-checker.english.md]: I have removed this 🙂

**Comment by RandellDawson** [comment]
@nhcarrigan When I test the example project against this PR, it fails the last test.

**Comment by nhcarrigan** [comment]
Ah, I believe the example project is missing the `.env` declaration `NODE_ENV=test`. I am not sure if that is intentional, but I [forked the example project](https://stock-price-checker.nicholascarriga.repl.co/) and included that value to get the necessary results to generate.

**Comment by RandellDawson** [comment]
Fixed merge conflict, and PR #39024 has been merged, so removing `status: blocked` label.

**Review by moT01 — APPROVED** [review]
LGTM 🎉 We can adjust the user stories with another PR for the issue about those.

**Review by RandellDawson — APPROVED** [review]
LGTM

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**